mtenn/conversion_utils/visnet.py

```diff
@@ -0,0 +1,237 @@
+"""
+Representation and strategy for ViSNet model.
+"""
+from copy import deepcopy
+import torch
+from torch.autograd import grad
+from torch_geometric.utils import scatter
+
+from mtenn.model import GroupedModel, Model
+from mtenn.strategy import ComplexOnlyStrategy, ConcatStrategy, DeltaStrategy
+
+HAS_VISNET = False
+try:
+    from torch_geometric.nn.models import ViSNet as PygVisNet
+    HAS_VISNET = True
+except ImportError:
+    pass    
+if HAS_VISNET:
+    class EquivariantVecToScaler(torch.nn.Module):
```

Comment:
Techically this can live outside the guard, also do you mean `Scalar` or `Scaler`?